### DISCUSSION TITLE
[Request] Supabase

### DISCUSSION THREAD

**Discussion by fguisso** [tteck/Proxmox#3009] [Ideas]
The open source Firebase alternative. 
- [x] Hosted Postgres Database. [Docs](https://supabase.com/docs/guides/database)
- [x] Authentication and Authorization. [Docs](https://supabase.com/docs/guides/auth)
- [x] Auto-generated APIs.
  - [x] REST. [Docs](https://supabase.com/docs/guides/api)
  - [x] GraphQL. [Docs](https://supabase.com/docs/guides/graphql)
  - [x] Realtime subscriptions. [Docs](https://supabase.com/docs/guides/realtime)
- [x] Functions.
  - [x] Database Functions. [Docs](https://supabase.com/docs/guides/database/functions)
  - [x] Edge Functions [Docs](https://supabase.com/docs/guides/functions)
- [x] File Storage. [Docs](https://supabase.com/docs/guides/storage)
- [x] AI + Vector/Embeddings Toolkit. [Docs](https://supabase.com/docs/guides/ai)
- [x] Dashboard

![Supabase Dashboard](https://raw.githubusercontent.com/supabase/supabase/master/apps/www/public/images/github/supabase-dashboard.png)

**Comment by bramsuurdje** [comment]
Supabase uses Docker under the hood, so we can't create a script for it. You can easily install it yourself, though. Just install a [Docker LXC](https://helper-scripts.com/scripts?id=Docker) and follow the [official guide](https://supabase.com/docs/guides/self-hosting/docker) on Supabase.

**Reply by tteck** [reply]
Or, use [Pocketbase](https://helper-scripts.com/scripts?id=Pocketbase) 😜

**Reply by fguisso** [reply]
I'll take a look in Pocketbase, but my goal with supabase is to use the Edge Functions. Any other options for that?
